### PR TITLE
Add Discord Links

### PR DESCRIPTION
**Pull request by mauerbac**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # AWS SDK for Android 
 
+[![DiscordChat](https://img.shields.io/discord/308323056592486420?logo=discord")](https://discord.gg/jWVbPfC)
```
Comment (jpignata) [inline README.md:3]: Looks like you have a stray quote mark in here, so the logo isn't rendering.

File: README.md [hunk 2]
```diff
@@ -190,7 +191,8 @@ For more information on Android development, see the Android developer site at:
 
 ## Talk to Us
 
-Visit our GitHub [Issues](https://github.com/aws-amplify/aws-sdk-android/issues) to leave feedback and to connect with other users of the SDK.
+* Visit our GitHub [Issues](https://github.com/aws-amplify/aws-sdk-android/issues) to leave feedback and to connect with other users of the SDK.
```
Comment (jpignata) [inline README.md:194]: Inconsistent punctuation across these two bullets.

File: README.md [hunk 3]
```diff
@@ -190,7 +191,8 @@ For more information on Android development, see the Android developer site at:
...
+* Join our [Discord Community](https://discord.gg/jWVbPfC)
```
Comment (jpignata) [inline README.md:195]: In the above we're linking "Issues" and not "GitHub", but here we're linking "Discord" and "Community." I prefer the latter, but please make these consistent.

File: README.md (hunk 2 above)
Comment (jpignata) [reply README.md:194]: Is it to leave feedback and connect with other users? This is a strange bullet. I think we need more prescriptive guidance on when to open an issue (bugs, problems) versus when to go to Discord (to connect with... etc).

**Comment by jpignata** [comment]
Please make sure your pull requests use the template.

**Comment by jpignata** [comment]
Please open a similar pull for https://github.com/aws-amplify/amplify-android.

File: README.md (hunk 2 above)
Comment (mauerbac) [reply README.md:194]: These are good points. For now I'll add the badge at the top and once we have more clarity on GitHub Issues vs Discussions vs Discord we can add links here.

**Pull request merged**